corehq/apps/app_manager/views.py

```diff
@@ -1563,6 +1569,21 @@ def rename_action_case_type(mod):
         )
         module.case_list_form.media_audio = val
 
+    if should_edit('case_list-menu_item_media_image'):
+        val = _process_media_attribute(
+            'case_list-menu_item_media_image',
+            resp,
+            request.POST.get('case_list-menu_item_media_image')
```

Comment:
i think this will raise an error if `case_list-menu_item_media_image` doesn't exist in the POST request. just wanted to check that you intend for that